lib/ex_aws/s3/impl.ex

```diff
@@ -232,6 +234,14 @@ defmodule ExAws.S3.Impl do
     body_binary = body |> IO.iodata_to_binary
 
     request(client, :post, bucket, "/?delete", body: body_binary, headers: %{"content-md5" => content_md5})
+    |> case do
+      {:ok, body} -> 
+        case tail == [] do
+          true -> {:ok, body}
+          false -> delete_multiple_objects(client, bucket, tail, opts)
```

Comment:
so options
Happy path
{:ok, number deleted} or {:ok, enumerable of items deleted}

error path
{:error, number deleted, error message} or  {:error, enumerable of items deleted, error message}